inspirehep/modules/workflows/models.py

```diff
@@ -26,12 +26,20 @@
 
 from datetime import datetime
 
+import enum
+
 from sqlalchemy.dialects import postgresql
 from sqlalchemy_utils.types import JSONType, UUIDType
 
 from invenio_db import db
 
 
+class SourceEnum(enum.IntEnum):
+    arxiv = 1
```

Comment:
Using strings would be clearer, like `arxiv = 'arxiv'`. Not sure if this is possible though.

Reply:
This did not work because there was no total order defined on just string values and the db needed it for some backstage operations.